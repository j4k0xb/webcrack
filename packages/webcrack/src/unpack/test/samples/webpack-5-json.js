(window['webpackJsonp'] = window['webpackJsonp'] || []).push([
  [],
  [
    function (module, exports, __webpack_require__) {
      var json = __webpack_require__(1);
      console.log(json);
    },
    function (module, exports) {
      module.exports = { foo: 'bar' };
    },
  ],
]);
