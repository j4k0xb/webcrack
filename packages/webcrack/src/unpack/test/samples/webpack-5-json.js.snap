WebpackBundle {
  "entryId": "",
  "modules": Map {
    "0" => WebpackModule {
      "ast": var json = require("./1.js");
console.log(json);,
      "id": "0",
      "isEntry": false,
      "path": "./0.js",
    },
    "1" => WebpackModule {
      "ast": module.exports = {
  foo: "bar"
};,
      "id": "1",
      "isEntry": false,
      "path": "./1.js",
    },
  },
  "type": "webpack",
}